.travis.yml

```diff
@@ -1,11 +1,18 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.10
-  - 2.2.6
-  - 2.3.3
-  - 2.4.0
+  # - 1.9.3
+  # - 2.0.0-p648
```

Comment:
i guess this means dropping support for ruby 1.x and 2.0.x /shrug
v 1.9.3 and 2.0.0 gave me problems. In travis, rvm install couldn't find these versions for some reason. I didn't try particular long to try to resolve that. given that they're very old versions, i just commented it out and went on my happy way

i guess this really only matters if we think we're going to want to merge this back into the main repo.

Reply:
Huh, yeah, I think it's reasonable to drop support for those versions now.

The original repo has been archived, so we don't have to worry about merging this back in.